### DISCUSSION TITLE
Where are the DNS files or data?

### DISCUSSION THREAD

**Discussion by citycritter** [mvt-project/mvt#419] [Q&A] [answered]
Where would I view DNS data using MVT?

**Comment by Te-k** [comment, marked as answer]
Hi,

As far as I know, DNS logs aren't stored on phones so you won't be able to get them through MVT. The best way to get that info is most probably to use a custom DNS resolver or a wifi router that allows to capture network traffic (like the [Pirogue](https://pts-project.org/))
